database/migrations/2025_12_08_145911_update_leaderboarddef_status.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    public function up(): void
+    {
+        Schema::table('LeaderboardDef', function (Blueprint $table) {
+            $table->string('state')->default('active')->after('DisplayOrder')->index();
+        });
+    }
+
+    public function down(): void
+    {
+        Schema::table('LeaderboardDef', function (Blueprint $table) {
+            $table->dropIndex('state');
```

Comment:
This down migration fails.

The index was created without a name, but `dropIndex('state')` is looking for an index named exactly "state".